english/coffee/procrastinate/procrastinate.md

```diff
@@ -2,4 +2,5 @@ As you sit there and sip your coffee pondering over this huge marshmallow proble
 you strike upon a golden idea! Why hadn't you thought of this before?
 Procrastinate!! Well, That seems to work every time you want to avoid a problem.
 You flip out your phone and tap on that little twitter icon and voila!
-Problems just disappear and you are in a whole new world!
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.